generation/diff_files.sh

```diff
@@ -4,20 +4,27 @@
 # not using set -e because failing diff command is ok
 # latest repositories Id -> comgooglecloud-5570, comgoogleapi-5038, comgoogleanalytics-1052
 # this script must be run on a branch which does not have any snapshot versions in it
-# Run the stage job for google-cloud-java, on a branch which does not have any snapshot versions in it.
-# search the stage job logs for the latest-repo-ids and edit them below under `cloudRepoId`, `apiRepoId`, `analyticsRepoId`
+
+## HOW TO USE THIS SCRIPT ##
+# 1. Run the stage job for google-cloud-java, on a branch which does not have any snapshot versions in it.
```

Comment:
Does mean mean we merge the Release Please pull request? Or manually via Fusion. (Update source code comment)

Reply:
We can merge the release-please pull request when it has all the modules in it, right now it does not. Doing it manually via fusion on a branch which does not have any -snapshot version in it is the only way we have right now.

Reply:
Update source code comment